pkgs/by-name/_6/_64gram/package.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, fetchFromGitHub
+, telegram-desktop
+}:
+
+telegram-desktop.overrideAttrs (old: rec {
+
+  pname = "64Gram";
+  version = "1.1.15";
+
+  src = fetchFromGitHub {
+    owner = "TDesktop-x64";
+    repo = "tdesktop";
+    rev = "v${version}";
+
+    fetchSubmodules = true;
+    hash = "sha256-3HLRv8RTyyfnjMF7w+euSOj6SbxlxOuczap5Nlizsvg=";
+  };
+
+  meta = with lib; {
+    description = "64Gram (unofficial Telegram Desktop)";
```

```suggestion
    description = "An unofficial Telegram Desktop providing Windows 64bit build and extra features";
```

The description should make sense, but does not necessarily need to be taken from the repository description.